src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/UploadFileItem.vue

```diff
@@ -64,6 +70,11 @@ export default defineComponent({
       type: Number,
       required: true,
     },
+    canDelete: {
+      type: Boolean,
+      required: false,
+      default: true,
+    },
```

Comment:
I think the requirement was that uploaded files can be deleted, 

however existing files cannot be deleted. 

This reads like all files cannot be deleted.